src/mapbuffer.h

```diff
@@ -9,6 +9,7 @@
 
 #include "coords_fwd.h"
 #include "point.h"
+#include "multiworld.h"
```

Comment:
These should be going away anyway since multiworld.h should go away, but having a header added to a header is a bad practice, it should be added to the .cpp file where multiworld methods are referenced instead.